public/index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../src/styles/reset/reset.css">
+    <!-- <link rel="stylesheet" href="CSS/style.css"> -->
+    <link rel="stylesheet" href="/src/styles/sass/main.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="header__container container">
+            <h1 class='header__logo'>
+                <img
+                  src='./assets/logo.svg'
+                  alt="logo"
+                >
+                </img>
+              </h1>
+            <div class="navigation"></div>
+        </div>
+        <div class="hero container" id="hero" >
+            <div class="hero__info">
+                <h2 class="hero__header">Beautiful Free Nova template</h2>
+                <p class="hero__sub-header">A top notch premium quality template for your next web project</p>
+                <button class="hero__button">Download FREE!</button>
+            </div>
+            <div class="hero__photo-container"> 
+                <img class="image" 
+                    src="../images/screen.png"
+                    alt="screen"
+                    width="350px">
+                
+            </div>
+        </div>
+    </header>
+    <main> 
+        <section class="features" id="features">
+            <h2 class="features__header container">FEATURES</h2>
+               <div class="features__container container"></div>
```

Comment:
Jeśli tworzymy `<div>` tylko na potrzeby "tła/grafiki" to można wykorzystać `::after/::before`